Readme.md

```diff
@@ -11,18 +11,18 @@ Demo Orchard Core module for training purposes guiding you to become an Orchard
 
 ## How to start
 
-You can use this module as part of a vanilla Orchard Core source that includes the full source code - which is the recommended way. You can also use it as part of a solution that uses Orchard Core NuGet packages, however, it's harder to look under the hood of Orchard Core features.
+You can use (and are recommended to use) this module as part of a vanilla Orchard Core source that includes the full source code. You can also use it as part of a solution that uses Orchard Core NuGet packages. Note that it is harder to inspect Orchard Core features.

```

Comment:
I miss from the last sentence that it refers to the approach of going with NuGet packages.